File: 0001 Code Chef/0002 Mango Lassi/implementation/0003 c#/MangoLassi.cs
using System;

class MangoLassi{
    static void Main(string[] args){
        string stringInput = Console.ReadLine();
        int temperature = Convert.ToInt32(stringInput);
        if(temperature > 35){
            Console.WriteLine("YES");
        }else{
            Console.WriteLine("NO");
        }
    }
}

File: 0001 Code Chef/0002 Mango Lassi/implementation/0005 kotlin/MangoLassi.kt
fun main(args:Array<String>){
    var stringInput:String? = readLine();
    if(stringInput!=null){
        var day:Int? = stringInput.toIntOrNull();
        if(day!=null){
            if(day > 35){
                println("YES");
            }else{
                println("NO");
            }
        }
    }
}